### PR TITLE
SRVKE-573- Add net I/O in serving dashboard

### PR DESCRIPTION
**Pull request by skonto**
image

image

### CODE REVIEW

File: knative-operator/deploy/resources/dashboards/grafana-dash-knative.yaml [hunk 1]
```diff
@@ -1090,7 +1090,199 @@ data:
...
+             {
+                "expr": "sum(rate(container_network_receive_bytes_total{namespace=\"$namespace\", container=\"POD\", name=~\"k8s_POD_$deployment.*\"}[1m]))",
+                "format": "time_series",
+                "intervalFactor": 1,
+                "legendFormat": "{{received bytes}}",
+                "refId": "A"
+              },
+              {
+                "expr": "sum(rate(container_network_transmit_bytes_total{namespace=\"$namespace\", container=\"POD\", name=~\"k8s_POD_$deployment.*\"}[1m]))",
+                "legendFormat": "{{transmitted bytes}}",
```
Comment (markusthoemmes) [inline knative-operator/deploy/resources/dashboards/grafana-dash-knative.yaml:1144]: This seems to appear with the curly braces in the output. Should it?

File: knative-operator/deploy/resources/dashboards/grafana-dash-knative.yaml [hunk 2]
```diff
@@ -1090,7 +1090,199 @@ data:
...
+            "seriesOverrides": [],
+            "spaceLength": 10,
+            "stack": false,
+            "steppedLine": false,
+            "targets": [
+             {
+                "expr": "sum(rate(container_network_receive_bytes_total{namespace=\"$namespace\", container=\"POD\", name=~\"k8s_POD_$deployment.*\"}[1m]))",
+                "format": "time_series",
+                "intervalFactor": 1,
+                "legendFormat": "{{received bytes}}",
```
Comment (markusthoemmes) [inline knative-operator/deploy/resources/dashboards/grafana-dash-knative.yaml:1139]: ditto.

**Comment by markusthoemmes** [comment]
Is this something we want to do for the admin dashboards? I thought the service specific stuff would have to wait for OCP features. I might be mixing something up though.

File: knative-operator/deploy/resources/dashboards/grafana-dash-knative.yaml (hunk 2 above)
Comment (skonto) [reply knative-operator/deploy/resources/dashboards/grafana-dash-knative.yaml:1139]: I will remove its just an incompatibility between native dashboards and Grafana.

**Comment by skonto** [comment]
@markusthoemmes this dashboard is already deployed because it does need the queue proxy endpoint, it is just using the kube metrics available already. It is just metrics from a pod perspective.

**Review by markusthoemmes — APPROVED** [review]
/lgtm
/approve

**Pull request merged**